tests/src/common/find_service_setup.rs

```diff
@@ -33,34 +32,26 @@ pub async fn retry_client(url: &'static str) -> pubsub::find_service::Client {
 }
 
 impl FindService {
-    pub async fn new(_handle: tokio::runtime::Handle) -> FindService {
+    pub async fn new() -> FindService {
+        info!("Starting new meetme service");
         let path = "../target/debug/pubsub-meetup";
         let url = "http://127.0.0.1:8080/";
         let bind = "127.0.0.1:8080";
 
-        let _proc = _handle.enter(|| {
-            info!("Starting meetme service");
-            let ret =
-                tokio::process::Command::new(path)
-                    .arg("--bind")
-                    .arg(bind)
-                    .kill_on_drop(true)
-                    .spawn()
-                    .expect("Failed to start meetup");
+        let _proc = tokio::process::Command::new(path)
+            .arg("--bind")
+            .arg(bind)
+            .kill_on_drop(true)
+            .spawn()
+            .expect("Failed to start meetup");
 
-            info!("meetme service started");
-            //tokio::runtime::Handle::current().block_on(tokio::time::delay_for(std::time::Duration::from_secs(1)));
-            //info!("meetme service started");
-            ret
-        });
+        info!("meetme service started");
+        //tokio::runtime::Handle::current().block_on(tokio::time::delay_for(std::time::Duration::from_secs(1)));
```

Comment:
Could you remove these comment of mine? They should not have been included like this.